ocs_ci/ocs/ui/block_pool.py

```diff
@@ -173,3 +186,33 @@ def reach_pool_limit(self, replica, compression):
                     raise PoolStateIsUnknow(
                         f"pool {pool_name} is in unexpected state {pool_state}"
                     )
+
+    def pool_raw_capacity_loaded(self, pool_name):
+        """
+        Takes pool name and returns True if the raw capacity of the block pool is loaded
+        or returns False if the capacity is not loaded.
+
+        Args:
+            pool_name (str): The name of the pool to be deleted
+
+        Returns:
+            bool: True if raw capacity of the blockpool is loaded, otherwise False
+        """
+        logger.info("Checking if the Block Pool Raw Capacity has loaded in UI")
+        self.navigate_overview_page()
+        self.navigate_block_pool_page()
+        self.page_has_loaded()
+        if self.ocp_version_full <= version.VERSION_4_8:
+            self.do_click((f"{pool_name}", By.LINK_TEXT))
+        else:
+            self.do_click((f"//a[text()='{pool_name}']", By.XPATH))
+        time.sleep(5)
+        raw_capacity_loaded = self.check_element_text(
```

Comment:
it is good to check if the raw capacity shown in UI is indeed the correct value?

Reply:
@keesturam I have added this test in the latest commit